sdk/storage/inc/blobs/protocol/blob_rest_client.hpp

```diff
@@ -8176,6 +8222,73 @@ namespace Azure { namespace Storage { namespace Blobs {
             std::move(response), std::move(pHttpResponse));
       }
 
+      struct SealAppendBlobOptions
+      {
+        Azure::Core::Nullable<int32_t> Timeout;
+        Azure::Core::Nullable<std::string> LeaseId;
+        Azure::Core::Nullable<std::string> IfModifiedSince;
+        Azure::Core::Nullable<std::string> IfUnmodifiedSince;
+        Azure::Core::Nullable<std::string> IfMatch;
+        Azure::Core::Nullable<std::string> IfNoneMatch;
+        Azure::Core::Nullable<int64_t> AppendPosition;
+      }; // struct SealAppendBlobOptions
+
+      static Azure::Core::Response<SealAppendBlobResult> Seal(
+          const Azure::Core::Context& context,
+          Azure::Core::Http::HttpPipeline& pipeline,
+          const std::string& url,
+          const SealAppendBlobOptions& options)
+      {
+        unused(options);
```

Comment:
It's time to delete this right?

Reply:
I cannot remember which one, but some API doesn't use `options` at all